modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/classvars.mustache

```diff
@@ -53,8 +53,20 @@
 {{/minimum}}
 {{#pattern}}
             'regex': {
+            {{#isBinary}}
                 'pattern': r'{{{vendorExtensions.x-regex}}}',  # noqa: E501{{#vendorExtensions.x-modifiers}}
                 {{#-first}}'flags': (re.{{.}}{{/-first}}{{^-first}}          re.{{.}}{{/-first}}{{^-last}} | {{/-last}}{{#-last}}){{/-last}}{{/vendorExtensions.x-modifiers}}
+            {{/isBinary}}
+            {{^isBinary}}
+                {{#isByteArray}}
+                'pattern': r'{{{vendorExtensions.x-regex}}}',  # noqa: E501{{#vendorExtensions.x-modifiers}}
+                {{#-first}}'flags': (re.{{.}}{{/-first}}{{^-first}}          re.{{.}}{{/-first}}{{^-last}} | {{/-last}}{{#-last}}){{/-last}}{{/vendorExtensions.x-modifiers}}
+                {{/isByteArray}}
+                {{^isByteArray}}
+                'pattern': r'{{{vendorExtensions.x-regex}}}',  # noqa: E501{{#vendorExtensions.x-modifiers}}
+                {{#-first}}'flags': (re.{{.}}{{/-first}}{{^-first}}          re.{{.}}{{/-first}}{{^-last}} | {{/-last}}{{#-last}}){{/-last}}{{/vendorExtensions.x-modifiers}}
+                {{/isByteArray}}
+            {{/isBinary}}
```

Comment:
Is this working?
I am not seeing any samples have binary regex patterns.

Reply:
@spacether Without any of the other `bytes` stuff, this isn't actually needed anymore. It's another thing kept for the sake of the next PR (so the logic/need isn't forgotten).

Reply:
When we separate pieces of code like this, I worry that we will forget to include tests or samples that demonstrate them working.

- If you have it here can you add a spec example which creates binary regex patterns in the python-experimental sample?
- Otherwise please remove this from here and include it in your next PR